src/Sentry/Internal/Hub.cs

```diff
@@ -360,9 +409,31 @@ SentryId IHubEx.CaptureEventInternal(SentryEvent evt, Hint? hint, Scope? scope)
                 actualScope.SessionUpdate = _sessionManager.ReportError();
             }
 
-            // When a transaction is present, copy its DSC to the event.
-            var transaction = actualScope.Transaction as TransactionTracer;
-            evt.DynamicSamplingContext = transaction?.DynamicSamplingContext;
+            TransactionTracer? transaction = null;
+            if(_options.IsTracingEnabled)
+            {
+                // Inject trace information from a linked span
+                if (GetLinkedSpan(evt, actualScope) is { } linkedSpan)
+                {
+                    evt.Contexts.Trace.SpanId = linkedSpan.SpanId;
+                    evt.Contexts.Trace.TraceId = linkedSpan.TraceId;
+                    evt.Contexts.Trace.ParentSpanId = linkedSpan.ParentSpanId;
+                }
+
+                // When a transaction is present, copy its DSC to the event.
```

Comment:
This isn't super helpful, but I suggested comments above could. To clarify: Maybe it's my lack of understanding of `IsTracingEnabled`. Is this meaning `IsSentryPeformanceMonitoringWithTrasnactions` instead? Because what both sides of the `if` do are tracing

Reply:
Yes. Effectively, the scope  now contains a traceID in its `PropagationContext`. This turns `IsTracingEnabled` into `IsSentryPeformanceMonitoringWithTrasnactions`.
This also means that `Tracing without Performance` is not controlled by any option.

Reply:
Just checking my understanding:
* `IsTracingEnabled == true` => Tracing: `Enabled`; Sentry Performance Monitoring: `Enabled`
* `IsTracingEnabled == false` => Tracing: `Enabled`; Sentry Performance Monitoring: `Disabled`

Is that correct? If so then Tracing is now always enabled and that flag in the options probably needs to be renamed to be less confusing (maybe `IsPerformanceMonitoringEnabled`). We could mark the old one as obsolete and introduce a new one instead.

Reply:
OK, I think worth doing that in this pull request then... it's certain to cause confusion otherwise.